### ISSUE TITLE
telegraf fails to launch if tagexclude is added to http output plugin

### ISSUE THREAD

**Issue by niftyhoot** [influxdata/telegraf#5804]
### Relevant telegraf.conf:

```
[[outputs.http]]
  url = "http://35.192.104.17/ingest"
  tagexclude = ["xxx"]
```

### System info:
Telegraf version 1.10.3
Ubuntu 16.04

### Steps to reproduce:

1. stick in a tagexclude keyword in http output plugin

### Expected behavior:
http output should filter out specified tag

### Actual behavior:
Telegraf crashes
### Additional info:
```
/usr/bin/telegraf --config /etc/telegraf/telegraf.conf --config-directory /etc/telegraf/conf.d
2019-05-06T04:02:36Z I! Starting Telegraf 1.10.3
panic: reflect.MakeSlice of non-slice type

goroutine 1 [running]:
reflect.MakeSlice(0x26b8640, 0x1dcdd40, 0x0, 0x1, 0x1dcdd40, 0x1d648a0, 0xc000204b60)
	/usr/local/go/src/reflect/value.go:2122 +0x1e2
github.com/influxdata/telegraf/vendor/github.com/influxdata/toml.setArray(0x1dcdd40, 0xc000204b60, 0x198, 0xc0003d5700, 0xc, 0x0)
	/go/src/github.com/influxdata/telegraf/vendor/github.com/influxdata/toml/decode.go:347 +0x213
github.com/influxdata/telegraf/vendor/github.com/influxdata/toml.setValue(0x1dcdd40, 0xc000204b60, 0x198, 0x2694940, 0xc0003d5700, 0x1dcdd40, 0xc000204b60)
	/go/src/github.com/influxdata/telegraf/vendor/github.com/influxdata/toml/decode.go:266 +0x47c
github.com/influxdata/telegraf/vendor/github.com/influxdata/toml.UnmarshalTable(0xc0003860a0, 0x1e78440, 0xc0003776b0, 0x1e78440, 0xc0003776b0)
	/go/src/github.com/influxdata/telegraf/vendor/github.com/influxdata/toml/decode.go:127 +0xfbc
github.com/influxdata/telegraf/vendor/github.com/influxdata/toml.UnmarshalTable(0xc000386050, 0x2186520, 0xc0001063c0, 0xc00004e420, 0x0)
	/go/src/github.com/influxdata/telegraf/vendor/github.com/influxdata/toml/decode.go:161 +0xa56
github.com/influxdata/telegraf/internal/config.(*Config).addOutput(0xc0003708c0, 0xc000374298, 0x4, 0xc000386050, 0x0, 0x0)
	/go/src/github.com/influxdata/telegraf/internal/config/config.go:874 +0x133
github.com/influxdata/telegraf/internal/config.(*Config).LoadConfig(0xc0003708c0, 0xc000054390, 0x2b, 0xc000000300, 0xc00005c000)
	/go/src/github.com/influxdata/telegraf/internal/config/config.go:662 +0xc07
github.com/influxdata/telegraf/internal/config.(*Config).LoadDirectory.func1(0xc000054390, 0x2b, 0x26a6560, 0xc000190820, 0x0, 0x0, 0x16, 0xc0003fb898)
	/go/src/github.com/influxdata/telegraf/internal/config/config.go:553 +0x167
path/filepath.walk(0xc000054390, 0x2b, 0x26a6560, 0xc000190820, 0xc000205c00, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:358 +0x41c
path/filepath.walk(0x7fff824527d9, 0x14, 0x26a6560, 0xc0001915f0, 0xc000205c00, 0x0, 0x10)
	/usr/local/go/src/path/filepath/path.go:382 +0x2fe
path/filepath.Walk(0x7fff824527d9, 0x14, 0xc000205c00, 0x2265b01, 0xc00046a000)
	/usr/local/go/src/path/filepath/path.go:404 +0x105
github.com/influxdata/telegraf/internal/config.(*Config).LoadDirectory(0xc0003708c0, 0x7fff824527d9, 0x14, 0x0, 0x0)
	/go/src/github.com/influxdata/telegraf/internal/config/config.go:559 +0x6f
main.runAgent(0x269c340, 0xc0003d4e80, 0x3fb82f0, 0x0, 0x0, 0x3fb82f0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/influxdata/telegraf/cmd/telegraf/telegraf.go:128 +0xcfb
main.reloadLoop(0xc00009c180, 0x3fb82f0, 0x0, 0x0, 0x3fb82f0, 0x0, 0x0, 0xc000445d58, 0x0, 0x0, ...)
	/go/src/github.com/influxdata/telegraf/cmd/telegraf/telegraf.go:102 +0x267
main.main()
	/go/src/github.com/influxdata/telegraf/cmd/telegraf/telegraf.go:386 +0x4ba
```

**Comment by glinton** [comment]
This wasn't caused by adding `tagexclude` to the http output plugin alone. Can you provide the full config loaded by telegraf? Something like `cat /etc/telegraf/conf.d/*.conf` may do the trick.

**Comment by glinton** [comment]
I can reproduce this now, but it required a config as follows:
```toml
[[outputs.http]]
  url = ["http://x.x.x.x"]
  tagexclude = ["xxx"]
```

**Comment by glinton** [comment]
Closed by 32f0cb16f5ae8e66da4fbeee17e63200e5bb2640. This shouldn't happen with the nightlies and will be fixed officially in the 1.11 release
